tests/components/roborock/test_image.py

```diff
@@ -151,22 +152,25 @@ async def test_fail_to_save_image(
     caplog: pytest.LogCaptureFixture,
 ) -> None:
     """Test that we gracefully handle a oserror on saving an image."""
-    # Reload the config entry so that the map is saved in storage and entities exist.
+    await async_setup_component(hass, DOMAIN, {})
```

Comment:
why do we use `async_setup_component` here?

Reply:
I noticed this also, though was preserving existing behavior. I assume it was just copied from other tests that do this.

Updated this to setup the config entry instead.